src/index.html

```diff
@@ -11,7 +11,123 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
```

Comment:
general: fix it
<img width="1037" alt="Screenshot 2023-01-12 at 16 59 52" src="https://user-images.githubusercontent.com/53228259/212100857-a26957e8-02ab-45ca-a5dd-8db2dab230a1.png">